marverenic/Jockey
# Use target fragments instead of tag lookups

Completes [#164621769](https://www.pivotaltracker.com/story/show/164621769).

This fixes an issue where the sleep timer and multi-repeat value pickers would not work because they were unable to send values back to the calling fragment. This was failing because they were trying to look up a fragment with a specific tag, and the requesting fragment did not have a tag specified. This regression was probably introduced in 4c3985ba45a26ee620a3516c8c87893174b4f426.

Instead of adding a tag to the fragments, this commit fixes the issue by replacing the tag lookup with the built-in target fragment mechanism, which removes the requirement that fragments need to have a tag for the result to get delivered to them.

### Testing Steps
Verify that the multi-repeat and sleep timer settings can be enabled from the now playing page. Only setting this preference is affected, the actual behavior of these options has not changed.
